tools/group_humann2_uniref_abundances_to_GO/.shed.yml

```diff
@@ -0,0 +1,7 @@
+categories: 
```

Comment:
Can we please write GO in small letters in the folder name to stick to the tool name.